themes/godotengine/pages/download/osx.htm

```diff
@@ -33,10 +33,11 @@
     <a href="https://store.steampowered.com/app/404790">Godot from Steam</a> to work around this.
   </p>
   <p>
-    You can also get Godot with <a href="https://brew.sh/">Homebrew</a>.
+    You can also get Godot with <a href="https://brew.sh/">Homebrew</a> or <a href="https://www.macports.org">MacPorts</a>.
     Note that the version available on Homebrew isn't code-signed either.
```

Comment:
If MacPorts releases don't run out of the box due to Gatekeeper restrictions, I suggest amending this notice to avoid confusion:

```suggestion
    Note that the versions available on Homebrew and MacPorts aren't code-signed either.
```

Reply:
No, that change isn't necessary. Software released through MacPorts runs out of the box and is not affected by Gatekeeper restrictions, and doesn't need to be code-signed/notarized.